packages/web-components/src/components/ic-accordion-group/ic-accordion-group.css

```diff
@@ -5,6 +5,8 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  --ic-typography-color: var(--ic-accordion-title-text);
+  color: var(--ic-accordion-title-text);
```

Comment:
I think this token should be renamed to `ic-accordion-group-title-text` to make it more reflective of where the token is being applied